index.js

```diff
@@ -32,6 +32,13 @@ Cypress.Commands.add('percySnapshot', (name, options) => {
   name = name || cy.state('runnable').fullTitle();
 
   cy.then(async () => {
+    if (Cypress.config('isInteractive')) {
+      return cylog('Disabled in interactive mode', {
+        name,
+        details: 'use "cypress run" instead of "cypress open"'
```

Comment:
For my own 🎨 sanity, can we flip the order of these two?
```suggestion
        details: 'use "cypress run" instead of "cypress open"',
        name
```